tests/recorder/recorder.go

```diff
@@ -0,0 +1,233 @@
+package main
```

Comment:
Hey, created github.com/coryb/autoplay repo with the recorder code moved over, I added you as a contributor to that repo.  I have updated the tests.  

The travis tests seem to be failing, but it works on my system, so I will have to debug that one now...